.github/workflows/helm-rollback.yaml

```
@@ -0,0 +1,57 @@
+name: Helm Rollback
+on:
+  workflow_call:
+    inputs:
+      namespace:
+        description: "Kubernetes namespace to deploy resources in"
```

Nit: Would the wording "rollback" be more fitting here:

```suggestion
        description: "Kubernetes namespace scope for this rollback"
```
/